docs/index.md

```diff
@@ -74,6 +74,18 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.google.gms.google-services' // <- Add this line
 ```
 
+#### Using Firebase services in an emulator
+
+To use Firebase services in an emulator, you need to install the Google Play services SDK. From Android Studio:
```

Comment:
This is only partially true, it's a lot more subtle than this

In order to use *parts* of the underlying firebase-android-sdk, like auth, a device (or emulator) must have Play Services installed. Other parts that need it I think are things that rely on instance ids like messaging, and remote config. Even with those there is some movement to make those possible without play services since devices like Huawei won't have them going forward - in other words it's actually an area of active development.

Here's auth without play services for instance https://github.com/FirebaseExtended/auth-without-play-services

There are many parts that do not require play services though. Analytics and Firestore I think don't need it for instance.

Additionally, I don't think the SDK has anything to do with it, so I believe the following instructions to be inaccurate.

I believe the important thing is to use an *emulator* (not the SDK) that has Play APIs or Play Store installed. That's what has always made the difference for me at least, and could perhaps use a mention

Finally - I think the better location for this is in the tips and tricks area with the header for the tip being the actual error message - https://rnfirebase.io/faqs-and-tips

Reply:
Thanks for the tips! I've moved it to FAQs and also found another alternative as well. I left them both documented. Let me know if you think I should clarify when this happens more.